### ISSUE TITLE
Horizontal scrolling in the report table

### ISSUE THREAD

**Issue by anantakrishna** [almarklein/timetagger#438]
Currently there is no horizontal scrolling, and the descriptions are visually cut.

Use case: I want to skim through the report without exporting it, this is how I consume the report.

**Comment by almarklein** [comment]
Is the "save as pdf" an option for you? The behavior might differ for different browsers, but when I click it, the browser immediately opens the pdf.

**Comment by anantakrishna** [comment]
Yes, it works. Though just due to the fact that the browser is configured to save downloads into a predefined directory. Otherwise one would deal with a save file dialog.

So, this can be seen as a workaround. Though if you don't mind and if I find time, I'd try to propose a CSS change.

**Comment by TejeshAkula** [comment]
Hey Can I attempt to solve this issue?

**Comment by almarklein** [comment]
Sure!
